Edge to edge behavior on Android (#13099)
flutter/website: src/content/release/breaking-changes/default-systemuimode-edge-to-edge.md

[edge-to-edge mode]: {{site.android-dev}}/develop/ui/views/layout/edge-to-edge
[Android 15 release notes]: {{site.android-dev}}/about/versions/15/behavior-changes-15#edge-to-edge
[`SystemChrome.setEnabledSystemUIMode`]: {{site.api}}/flutter/services/SystemChrome/setEnabledSystemUIMode.html

## Migration guide

> To opt out of edge-to-edge on Android SDK 15 (for Android 15 only),

Reviewer: Nit:
```suggestion
To opt out of edge-to-edge on Android SDK 15 and below,
```
